packages/module/src/DetailsPageHeader/utils/ActionButtons.tsx

```diff
@@ -0,0 +1,66 @@
+import { Button, Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+import React from 'react';
+
+export interface ActionButtonProps {
+  /** Text rendered inside the button */
+  label: string;
+  /** A callback for when the action button is clicked */
+  callback: (event: React.MouseEvent) => void;
+  /** Adds button variant styles */
+  variant?: 'primary' | 'secondary' | 'tertiary' | 'link';
```

Comment:
Can we use the PF Buttom variant type? It should already have all that is required. Unless we want to omit some values.

Reply:
Updated.